src/main/java/org/cats/CatsApplication.java

```diff
@@ -31,6 +33,8 @@
 @SpringBootApplication
 public class CatsApplication {
 
+	private static URI catsV1URI=URI.create("https://jsonplaceholder.typicode.com/posts");
```

Comment:
That's a good point. I actually thought we could configure it later when the api is ready. Put that for testing n didn't give it much thought.